cidc_api/models.py

```diff
@@ -482,6 +482,7 @@ def create(
         gcs_xlsx_uri: str,
         session: Session,
         commit: bool = True,
+        send_email: bool = False,
```

Comment:
shouldn't default be `True`?

Reply:
Since the email alert is an unexpected side-effect of this method, my preference is towards `False`.